sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_devices_first_seen_v1/metadata.yaml

```diff
@@ -0,0 +1,29 @@
+---
+friendly_name: FxA Users Services Devices First Seen
+description: Usage aggregations describing when each FxA device was first seen
```

Comment:
Maybe we can expand on the definition a little bit. See descriptions in [PR-3309](https://github.com/mozilla/bigquery-etl/pull/3309/files#)